src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/SimplifyBooleanExpressionCheck.java

```diff
@@ -48,18 +49,22 @@
  *     boolean a, b;
  *     Foo c, d, e;
  *
- *     if (!false) {}; // violation, can be simplified to true
+ *     if (!false) {};       // violation, can be simplified to true
  *
- *     if (a == true) {}; // violation, can be simplified to a
- *     if (a == b) {}; // OK
- *     if (a == false) {}; // violation, can be simplified to !a
+ *     if (a == true) {};    // violation, can be simplified to a
+ *     if (a == b) {};       // OK
+ *     if (a == false) {};   // violation, can be simplified to !a
  *     if (!(a != true)) {}; // violation, can be simplified to a
  *
  *     e = (a || b) ? c : d; // OK
- *     e = (a || false) ? c : d; // violation, can be simplified to a
+ *     e = (a || false) ? c : d;     // violation, can be simplified to a
```

Comment:
please revert this indentation changes, they are not related to issue and do not seem necessary

Reply:
The examples look better now imo.
Before - https://checkstyle.sourceforge.io/config_coding.html#SimplifyBooleanExpression
After - https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/8b38dbc_2021053600/config_coding.html#SimplifyBooleanExpression
https://github.com/checkstyle/checkstyle/pull/9136#discussion_r571714853

Reply:
minor commit is ok